packages/api-tests/test/window.spec.js

```diff
@@ -112,7 +114,7 @@ describe('Window API', function(done) {
 
     });
 
-    it('Should get the bounds of the window #ssf.Window.getBounds', function() {
+    it('Should get the bounds of the window @no-browser #ssf.Window.getBounds', function() {
```

Comment:
Nitpick: adding `@no-browser` means that there are now two ways of conditionally running the tests: using the `MOCHA_CONTAINER` env var, or using this (`@no-browser`) string in the test description, I'm not sure if there's a preference, but if we do stick if with the two different ways then maybe the window core tests can be grouped in a `describe` and all other tests can be grouped in a separate `describe` with `@no-browser` added? Or there could be separate spec files for the core and non-core window methods and these could be run based on the `MOCHA_CONTAINER` env var, which could simplify setup and reduce churn?